service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/dataproccluster/startup.sh

```diff
@@ -0,0 +1,788 @@
+#!/bin/bash
+#
+# Name: startup.sh
+#
+# NOTE FOR CONTRIBUTORS:
+#   This startup script closely mirrors the startup script used for Vertex AI Notebook instances here: /Users/rogerwangcs/local_dev/terra-workspace-manager/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh.
```

Comment:
Please remove `/Users/rogerwangcs/local_dev/` from the path.

Reply:
Ah, thanks for the catch